index.bs

```diff
@@ -2493,11 +2538,16 @@ To <dfn>obtain and deliver a debug report on source registration</dfn> given a
 
 1. If |dataType| is:
     <dl class="switch">
-    : "<code>[=source debug data type/source-destination-limit=]</code>"
-    :: [=map/Set=] |body|["`limit`"] to the user agent's [=max destinations covered by unexpired sources=],
-         [=serialize an integer|serialized=].
+    : "<code>[=source debug data type/source-success=]</code>"
+    : "<code>[=source debug data type/source-noised=]</code>"
+    :: If |destinationLimitHit| is true, [=map/set=] |body|["`source_destination_limit`"] to the user agent's [=max destinations covered by unexpired sources=],
+        [=serialize an integer|serialized=].
+
+    Note: The "`source_destiantion_limit`" field may be included to indicate that
+    [=max destinations covered by unexpired sources=] was hit.
```

Comment:
I think this note could use a bit more explanation. My immediate question on reading it is: "Why don't we just return a `source-destination-limit-hit` debug report instead of adding this field?" If it's to avoid leaking the exact error that occurred, we should say so, no?

Reply:
Elaborated a bit, PTAL, thanks.